nepalingo-web/src/hooks/Langauge.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useContext, useState, useEffect, createContext } from 'react';
+
+type ValueOf<T> = T[keyof T];
+export type Language = ValueOf<typeof Languages>
+
+export type LanguageKey = keyof typeof Languages;
+
+export const Languages = {
+    "Newari": "newari",
+    "Tajpuriya": "tajpuriya",
+    "Maithili": "coming soon",
+} as const
```

Comment:
why not just make it a string? this object type seems a bit redundant. Why is `as const` needed?